platform/security/handler/security-handler-basic/src/main/java/org/codice/ddf/security/handler/basic/WssBasicAuthenticationHandler.java

```diff
@@ -38,42 +36,63 @@
      */
     private static final String AUTH_TYPE = "WSSBASIC";
 
+    private final Parser parser;
+
+    /**
+     * Disable the default constructor
+     */
+    private WssBasicAuthenticationHandler() {
```

Comment:
You do not need to explicitly disable the private constructor once you have other constructors. The only time there is a default no-args ctor in Java is when no ctor has been declared.